components/Aside.vue

```diff
@@ -1,16 +1,32 @@
 <template>
 	<aside>
 		<ul>
-			<li v-for="h in headers" :key="h.id">
+			<!-- <li v-for="h in headers" :key="h.id">
 				<nuxt-link :to="h.href">
 					{{ h.title }}
 				</nuxt-link>
+			</li> -->
+			<li>
+				<nuxt-link to="#title1">
+					title1
+				</nuxt-link>
+			</li>
+			<li>
+				<nuxt-link to="#title2">
+					title2
+				</nuxt-link>
+			</li>
+			<li>
+				<nuxt-link to="#title3">
+					title3
+				</nuxt-link>
 			</li>
 		</ul>
 	</aside>
 </template>
 
 <script>
+// The above <li> are coded to show an example of how styles will look. This will be replaced by the commented out html code above.
```

Comment:
I think you should put the commented out code with the comment. Also I'm not sure why the commented out code isn't just being used. Please explain that in the comment.

Reply:
This was a personal preference. I don't really like how commented out HTML looks. Changing now